app/views/tracks/build/show.html.haml

```diff
@@ -1,6 +1,9 @@
+- track_meta_tags(@user_track)
```

Comment:
I've snuck this in as well

Reply:
Nice

Reply:
You'll actually need to change this, else we'll have duplicate meta-titles which hurt us from an SEO perspective.

So you'll need to manually do:

```
content_for :meta_title, "#{user_track.track_title} build page on Exercism"
```

etc (same for description, image, etc)

Reply:
I'll then remove it and fix it in another PR (other track pages have this too)